frontend/tests/ui/test_contest.py

```diff
@@ -116,13 +124,35 @@ def test_user_ranking_contest(driver):
                 (By.CSS_SELECTOR, '#ranking')))
 
         run_accepted_user = driver.browser.find_element_by_xpath(
-            '//td[contains(@class, "accepted")]/preceding-sibling::td[1]')
+            '//td[contains(@class, "accepted")]/preceding-sibling::td[@class='
+            '"user"]')
         assert run_accepted_user.text == user1, run_accepted_user
 
         run_wrong_user = driver.browser.find_element_by_xpath(
-            '//td[contains(@class, "wrong")]/preceding-sibling::td[1]')
+            '//td[contains(@class, "wrong")]/preceding-sibling::td[@class='
+            '"user"]')
         assert run_wrong_user.text == user2, run_wrong_user
 
+        contestants_full_list = driver.browser.find_elements_by_xpath(
+            '//*[@id="ranking"]/div/table/tbody/tr/td[@class="user"]')
+        contestants_set = {user1, user2, user3, driver.user_username}
+        contestants_full_set = set()
```

Comment:
esto puede ser un set comprehension (así como lo puse en la sugerencia):

```suggestion
        contestants_full_set = {element.text for element in contestants_full_list}
```

igual abajo.

Reply:
Listo, no había visto la sugerencia. Sorry